eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -469,13 +469,19 @@ constr<a> =
   ; similarly for tag range: 6.1280 .. 6.1400 inclusive
   / #6.102([uint, [* a]])
 
-redeemer = [ tag: redeemer_tag, index: uint, data: plutus_data, ex_units: ex_units ]
+redeemers =
+  [ + [ tag: redeemer_tag, index: uint, data: plutus_data, ex_units: ex_units ] ]
+; TODO: Add alternative implementation that reflects the reality more accuratly:
```

Comment:
Make sure there's an issue for this

Reply:
Thank you for keeping me honest :wink: 
There is already one #3922